templates/profile.html

```diff
@@ -16,42 +16,42 @@ <h2>About</h2>
               <a data-edit-name=socials-twitter href="{{ user.profile.socials.twitter }}" class=icon-twitter></a>
               <label for=social-twitter>
                   <span class=icon-twitter></span>
-                  <input type=url id=social-twitter>
+                  <input type=url id=social-twitter name=socials-twitter value="{{ user.profile.socials.twitter }}">
               </label>
           </li>
           <li>
               <a data-edit-name=socials-facebook href="{{ user.profile.socials.facebook }}" class=icon-facebook></a>
-              <label for=social-facebook>
+              <label for=socials-facebook>
                   <span class=icon-facebook></span>
-                  <input type=url id=social-facebook>
+                  <input type=url id=socials-facebook name=socials-facebook value="{{ user.profile.socials.facebook }}">
               </label>
           </li>
           <li>
               <a data-edit-name=socials-instagram href="{{ user.profile.socials.instagram }}" class=icon-instagram></a>
-              <label for=social-instagram>
+              <label for=socials-instagram>
                   <span class=icon-instagram></span>
-                  <input type=url id=social-instagram>
+                  <input type=url id=socials-instagram>
```

Comment:
missing `name` and `value`.
Same for fields below

Reply:
ok